submissions/YarHadi/html-css-popup/css/styles.css

```diff
@@ -0,0 +1,170 @@
+.nav-menu {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: rgb(51, 51, 51);
+  height: 60px;
+}
+
+.navmenu-link {
+  display: flex;
+  color: rgb(255, 255, 255);
+  text-align: center;
+  padding: 16px;
+  text-decoration: none;
+}
+
+.pop-input-ins,
+.pop-input {
+  position: absolute;
+  width: 0;
+  height: 0;
+}
+
+.pop-input-ins:focus ~ .more-btn {
+  background-color: rgba(0, 225, 255, 0.281);
+}
+
+.pop-input:focus ~ #nav-popup-btn-label {
+  background-color: rgba(0, 225, 255, 0.281);
+}
+
+.nav-item a:hover {
+  background-color: #111111;
+  cursor: pointer;
+}
+
+.nav-item a:focus {
+  background-color: rgba(0, 225, 255, 0.281);
+  outline: none;
+}
+
+.nav-item label {
+  display: flex;
+  color: rgb(255, 255, 255);
+  justify-content: center;
+  text-align: center;
+  padding: 16px;
+  text-decoration: none;
+}
+
+.nav-item label:hover {
+  background-color: #111111;
+  cursor: pointer;
+}
+
+.nav-item label:focus {
+  background-color: rgba(0, 225, 255, 0.281);
```

Comment:
pretty strange decisions of choosing the colors. why do you need alpha with such accuracy?